app/Console/Commands/RunScheduledJobsManually.php

```
@@ -23,37 +24,52 @@ class RunScheduledJobsManually extends Command
-        $scheduled_backups = ScheduledDatabaseBackup::where('enabled', true)->get();
+        $query = ScheduledDatabaseBackup::where('enabled', true);
+
+        if ($frequency) {
+            $query->where(function ($q) use ($frequency) {
+                // Handle human-readable frequency strings
+                if (in_array($frequency, ['daily', 'hourly', 'weekly', 'monthly', 'yearly', 'every_minute'])) {
+                    $q->where('frequency', $frequency);
+                } else {
+                    // Handle cron expressions
+                    $q->where('frequency', $frequency);
+                }
```

PHPMD (2.15.0)

The method runScheduledBackups uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)